core/src/main/scala/com/snowplowanalytics/snowplow/eventgen/protocol/Context.scala

```diff
@@ -20,10 +20,15 @@ import io.circe.syntax.EncoderOps
 import io.circe.{Encoder, Json, JsonObject}
 import org.apache.http.message.BasicNameValuePair
 import org.scalacheck.Gen
+import java.util.concurrent.atomic.AtomicLong
+
 
 object Context {
+
+  var changeFormGenCount = new AtomicLong()
```

Comment:
It's a nit, but I think these can be `val`s not `var`s.  Because you never re-assign to them.